### PR TITLE
[IMP] account,base: bank statement management

### PR DESCRIPTION
**Pull request by h4818**
[IMP] account: statement management 2.1
In the process of making bank statements optional, the usability of bank statement was reduced.
This PR aims improve that, by providing the ability to view and manage bank statements on the bank reconciliation widget.
See the Enterprise PR.

[IMP] base: force dialog size of layout configurator
action_base_document_layout_configurator should always have a dialog_size = 'large'
this ensures that the context from a calling function is not applied

Task-3270046

### CODE REVIEW

File: addons/account/models/account_bank_statement.py [hunk 1]
```diff
@@ -104,6 +105,11 @@ class AccountBankStatement(models.Model):
     # -------------------------------------------------------------------------
     # COMPUTE METHODS
     # -------------------------------------------------------------------------
+    @api.depends('create_date')
```
Comment (smetl) [inline addons/account/models/account_bank_statement.py:108]:
```suggestion

    @api.depends('create_date')
```
:roll_eyes:

File: addons/account/models/account_bank_statement.py [hunk 2]
```diff
@@ -104,6 +105,11 @@ class AccountBankStatement(models.Model):
...
+    def _compute_name(self):
+        for stmt in self:
+            stmt.name = _("%s Statement %s", stmt.journal_id.code, stmt.date)
```
Comment (smetl) [inline addons/account/models/account_bank_statement.py:111]: Are you not loosing the name when saving since `create_date` is set upon the creation

Comment (h4818) [reply addons/account/models/account_bank_statement.py:112]: Nope, it works as expected

**Review by smetl — APPROVED** [review]

**Comment by smetl** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**